Makefile

```diff
@@ -33,8 +33,16 @@ lint:
 
 .PHONY: acc-test
 acc-test:
+ifndef LIGHTSTEP_API_KEY_PUBLIC
+	$(error LIGHTSTEP_API_KEY_PUBLIC must be defined for acc-test)
+endif
```

Comment:
Added this as it is a little confusing that `LIGHTSTEP_API_KEY_PUBLIC` needs to be set, but the internal error says `LIGHTSTEP_API_KEY` needs to be set (since, per the next line, the environment variable is remapped before execution).